### PR TITLE
Create an account id when running the binary

### PR DESCRIPTION
**Pull request by TimDiekmann**
## 🌟 What is the purpose of this PR?

Currently, it's pretty hard to test the REST endpoints as no account id is generated and authentication is currently not implemented.

## 🔗 Related links

- [Asana task](https://app.asana.com/0/1202542409311090/1202637673613186/f) _(internal)_

## 🔍 What does this change?

- implement `Display` on `AccountId`
- Add method on `PostgresDatabase` to insert an account id (will be removed when authentication is in place)
- Create an account id when running the binary and output it on the terminal
- Add a `cargo-make` command to run these tests

## ⚠️ Known issues

This is a temporary workaround until authentication is in place but it's needed to allow simple manual testings for the REST API

## 📹 Demo

```
$ cargo run
  2022-07-20T13:44:16.875858Z DEBUG graph::datastore::postgres: Creating connection pool to Postgres
    at lib/graph/src/datastore/postgres/mod.rs:40

  2022-07-20T13:44:17.028639Z  INFO hash_graph: Created account id, account_id: 00000000-0000-0000-0000-000000000000
    at bin/hash_graph/src/main.rs:55

  2022-07-20T13:44:17.033651Z  INFO hash_graph: Listening on 127.0.0.1:3000
    at bin/hash_graph/src/main.rs:60
```

### CODE REVIEW

File: packages/graph/hash_graph/bin/hash_graph/src/main.rs [hunk 1]
```diff
@@ -35,6 +38,22 @@ async fn main() -> Result<(), GraphError> {
             err
         })?;
 
+    #[cfg(debug_assertions)]
+    let account_id = AccountId::new(Uuid::nil());
+    #[cfg(not(debug_assertions))]
+    let account_id = AccountId::new(Uuid::new_v4());
```
Comment (thehabbos007) [inline packages/graph/hash_graph/bin/hash_graph/src/main.rs:44]: While I do think this is a neat way to do this, I don't think we want to even create another UUID in release mode, and rather just use `Uuid::nil()` in either cases. This way we can still run our manual tests against release mode without having to change the UUID.

File: packages/graph/hash_graph/bin/hash_graph/src/main.rs [hunk 2]
```diff
@@ -35,6 +38,22 @@ async fn main() -> Result<(), GraphError> {
...
+
+    datastore
+        .insert_account_id(account_id)
+        .change_context(GraphError)
+        .await
+        .map_err(|err| {
+            tracing::error!("{err:?}");
+            err
+        })?;
```
Comment (thehabbos007) [inline packages/graph/hash_graph/bin/hash_graph/src/main.rs:53]: I don't think we want to short-circuit here as the binary won't ever start after the first time. We should ignore the result here if it's the duplicate foreign key error

**Review by thehabbos007 — CHANGES_REQUESTED** [review]
Thanks Tim, definitely useful! Just some things we should consider and a change request :)

File: packages/graph/hash_graph/bin/hash_graph/src/main.rs (hunk 2 above)
Comment (TimDiekmann) [reply packages/graph/hash_graph/bin/hash_graph/src/main.rs]: That's a good point

File: packages/graph/hash_graph/lib/graph/src/types/mod.rs [hunk 3]
```diff
@@ -1 +1,90 @@
+//! Model types used across datastores
```
Comment (thehabbos007) [inline packages/graph/hash_graph/lib/graph/src/types/mod.rs:1]: Not sure what happened in this, but the file at
https://github.com/hashintel/hash/blob/ec1a2fb665500256aa77b31bfc3bc0fb788835b1/packages/graph/hash_graph/lib/graph/src/ontology/mod.rs contains some of the same defs, but not the qualified

**Review by thehabbos007 — CHANGES_REQUESTED** [review]
I approve of the changes, but I think the module restructure made the rebase weird

File: packages/graph/hash_graph/lib/graph/src/types/mod.rs (hunk 3 above)
Comment (TimDiekmann) [reply packages/graph/hash_graph/lib/graph/src/types/mod.rs]: Definitely, my IDE tried to be smart 🙃

File: packages/graph/hash_graph/lib/graph/src/types/mod.rs [hunk 4]
```diff
@@ -1 +0,0 @@
-
```
Comment (TimDiekmann) [inline packages/graph/hash_graph/lib/graph/src/types/mod.rs:1]: This is an unreferenced module, probably a previous rebase error.

**Review by thehabbos007 — APPROVED** [review]

**Pull request merged**